crates/dbs-legacy-devices/src/lib.rs

```diff
@@ -10,6 +10,8 @@
 mod serial;
 pub use self::serial::*;
 
+#[cfg(target_arch = "x86_64")]
+mod cmos;
```

Comment:
This also needs a `pub use self::cmos::*;`. At the moment the new device is private.